.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,20 @@
-# Fixes/Closes
-
-<!-- In general, PRs should fix an existing issue on the repo. -->
-<!-- Please link to that issue here as "Closes #(issue-number)". -->
-Closes #
-
 # Description
-<!-- What does this pull request (PR) do? Why is it necessary? -->
-<!-- Tell us about your new feature, improvement, or fix! -->
-<!-- If your change includes user interface changes, please add an image, or an animation "An image is worth a thousand words!" -->
-<!-- You can use https://www.cockos.com/licecap/ or similar to create animations -->
-
-# References
-<!-- What resources, documentation, and guides were used in the creation of this PR? -->
+<!-- What does this pull request (PR) do? Is it a new feature, a bug fix,
+an improvement, or someting else? Why is it necessary? If relevant, please add
+a screenshot or a screen capture: "An image is worth a thousand words!" -->
```

Comment:
If we are trying to be concise, should "An image is worth a thousand words" stay?

Reply:
Fair enough :P We could put just that instead of the verbose explanation in fact... I just didn't want to squeeze *too much* and come off as dry, but we can definitely reduce this still.

Reply:
Yes I understand, I don't have a strong feeling either way so I leave to you.